### PR TITLE
NxGlobalFooter2 - RSC-1783

### PR DESCRIPTION
**Pull request by rpokorny**
https://sonatype.atlassian.net/browse/RSC-1783

This implements the new global footer that should be used in conjunction with the new global sidebar.

### CODE REVIEW

**Review by JasonDSwearingen — APPROVED** [review]
+1

File: gallery/src/styles/NxGlobalFooter2/NxGlobalFooter2Example.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  faArrowLeft,
+  faArrowRight,
+  faLink,
+  faQuestionCircle,
+  faBell,
+  faCog,
+  faUserCircle
+} from '@fortawesome/free-solid-svg-icons';
+
+//const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
```
Comment (benflodge) [inline gallery/src/styles/NxGlobalFooter2/NxGlobalFooter2Example.tsx:31]: Needed?

File: gallery/src/styles/NxGlobalFooter2/NxGlobalFooter2InnerSidebarExample.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  faArrowLeft,
+  faArrowRight,
+  faLink,
+  faQuestionCircle,
+  faBell,
+  faCog,
+  faUserCircle
+} from '@fortawesome/free-solid-svg-icons';
+
+//const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
```
Comment (benflodge) [inline gallery/src/styles/NxGlobalFooter2/NxGlobalFooter2InnerSidebarExample.tsx:32]: Is this comment needed?

**Review by benflodge — APPROVED** [review]
+1 looks good.

There are a number of commented out lines for
```
//const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
```
If this isn't needed it should be removed.

**Comment by rpokorny** [comment]
> If this isn't needed it should be removed.

Those were related to the `TODO` comments about the header.  Now that `NxGlobalHeader2` is merged into the feature branch, I've merged it into this branch as well and made those updates, including deleting the commented-out lines you saw.

**Pull request merged**